doc/devel/document.rst

```diff
@@ -1002,6 +1002,79 @@ subdirectory, but  :file:`galleries/users_explain/artists` has a mix of
 any ``*.rst`` files to a ``:toctree:``, either in the ``README.txt`` or in a
 manual ``index.rst``.
 
+
+.. _writing-user-guide:
+
+Write user guide explanations
```

Comment:
```suggestion
Write user guide documentation
```
While the user guide is explanation-style, I prefer  the general therm "documentation " here. "User guide explanations" made me wonder if there are other parts of the user guide.